src/quick-lint-js/error-formatter.h

```diff
@@ -59,10 +59,17 @@ class error_formatter {
   // void write_after_message(severity, const source_code_span &origin);
 
   enum class severity {
+    warning,
     error,
     note,
```

Comment:
Nit: Sort these alphabetically.
```suggestion
    error,
    note,
    warning,
```